### ISSUE TITLE
Server kicking me (the host) and my friend after selecting an origin

### ISSUE THREAD

**Issue by himbocity** [gamerpotion/DarkRPG#41]
Basically what the title says, I'm running a fabric server for 1.19.2 with the same fabric version the modpack is using and the same mods, we manage to join fine but once we select an origin (which takes a super long time to pop up at first) we get kicked after a few seconds

**Comment by gamerpotion** [comment]
> Basically what the title says, I'm running a fabric server for 1.19.2 with the same fabric version the modpack is using and the same mods, we manage to join fine but once we select an origin (which takes a super long time to pop up at first) we get kicked after a few seconds

Hi, you closed this, just checking this is resolved?
what server host are you using?

**Comment by himbocity** [comment]
> > Basically what the title says, I'm running a fabric server for 1.19.2 with the same fabric version the modpack is using and the same mods, we manage to join fine but once we select an origin (which takes a super long time to pop up at first) we get kicked after a few seconds
> 
> Hi, you closed this, just checking this is resolved? what server host are you using?

Yeah, sorry about that. Rather than using the server pack that was in the files (that I didn't see before I closed this), I added the mods directly from the modpack to the server mods folder. The server launched, but playing on it was impossible (getting the origins popup took very long and after selecting one it timed us out) but after properly using the server pack I think the issue has been solved.
